includes/SpecialRemovePII.php

```diff
@@ -133,7 +134,7 @@ public function validate( array $formData ) {
 			return Status::newFatal( 'centralauth-rename-doesnotexist' );
 		}
 
-		$oldCentral = CentralAuthUser::getInstanceByName( $formData['oldname'] );
+		$oldCentral = \CentralAuthUser::getInstanceByName( $formData['oldname'] );
```

Comment:
Way that? Thus just makes the current behaviour, as if no `use` statement was added.

Reply:
Based it on the only other change in the equivalent PR for our other extensions to see if it did anything to help it pass.